comp/core/workloadmeta/collectors/internal/podman/podman.go

```diff
@@ -63,7 +63,16 @@ func (c *collector) Start(_ context.Context, store workloadmeta.Component) error
 		return dderrors.NewDisabled(componentName, "Podman not detected")
 	}
 
-	c.client = podman.NewDBClient(config.Datadog.GetString("podman_db_path"))
+	dbPath := config.Datadog.GetString("podman_db_path")
```

Comment:
I think we can change `podman_db_path` to be empty default in code and then have a small step here if empty to `stat` in order `/var/lib/containers/storage/libpod/bolt_state.db` and `/var/lib/containers/storage/libpod/db.sql` and set the first matching in `podman_db_path`, aborting if none found.